events/tests/test_workflow.py

```diff
@@ -48,7 +80,9 @@ async def test_blacklist(PhabricatorMock, mock_taskcluster):
 
 
 @pytest.mark.asyncio
-async def test_publish_results_success_mode(PhabricatorMock, mock_taskcluster):
+async def test_publish_results_success_and_error_mode(
```

Comment:
Let's split this into multiple smaller tests instead of modifying the existing test, this way we know right away, in case of failure, what exactly is failing

Reply:
Cool I'm going to do 3 tests, lando on success, fail and general_fail.